src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj

```diff
@@ -21,7 +21,6 @@
     <PackageReference Include="GitHubActionsTestLogger" Version="1.3.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" PrivateAssets="All" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" PrivateAssets="All" />
-    <PackageReference Include="Particular.Packaging" Version="1.4.0" PrivateAssets="All" />
```

Comment:
This project produce a source package, so it does need the Particular.Packaging reference.